quetz/tasks/mirror.py

```diff
@@ -278,36 +290,22 @@ def _upload_package(file, channel_name, subdir):
             file.file.close()
 
 
-def initial_sync_mirror(
-    channel_name: str,
-    remote_repository: RemoteRepository,
-    arch: str,
-    dao: Dao,
-    pkgstore: PackageStore,
-    auth: authorization.Rules,
-    includelist: List[str] = None,
-    excludelist: List[str] = None,
-    skip_errors: bool = True,
-    use_repodata: bool = False,
-):
-    force = True  # needed for updating packages
-    logger.info(
-        f"Running channel mirroring {channel_name}/{arch} from {remote_repository.host}"
-    )
-
+def get_remote_repodata(
+    channel_name: str, arch: str, remote_repository: RemoteRepository
+) -> Union[dict, None]:
```

Comment:
Please add a docstring here that explains what the function does and most importantly how to interpret the return value (what does `None` mean?)